src/lib/constants.js

```diff
@@ -1,7 +1,14 @@
 const { BigNumber, ethers } = require('ethers');
 const { WeiPerEther } = ethers.constants;
 
+const ASSET_ID_MAPPING = {
+  0: { symbol: 'ETH', decimals: 18 },
+  1: { symbol: 'DAI', decimals: 18 },
+  6: { symbol: 'USDC', decimals: 6 },
+};
```

Comment:
@MilGard91 I've added the mapping in the constants.js file for simplicity. should it be in the store file initialState?

Reply:
Yes, sounds good if it not big of a refactor

Reply:
Also add `NXM`

Reply:
I've added decimals in the end in the store so there is one source of truth for assets info